src/Nethermind/Nethermind.Runner/configs/energyweb_archive.cfg

```diff
@@ -19,7 +19,6 @@
   "Metrics": {
     "NodeName": "Energy_Web Archive",
     "Enabled": false,
-    "PushGatewayUrl": "http://localhost:9091/metrics",
```

Comment:
Guess we need to do this for all other configs?

Reply:
Not sure. Probably?

Reply:
@LukaszRozmej wanna remove it or not?

Reply:
I'm gonna just merge this for now.